hack/tools.sh

```diff
@@ -33,11 +33,13 @@ declare -r CONTROLLER_TOOLS_VERSION=${CONTROLLER_TOOLS_VERSION:-v0.12.1}
 declare -r OPERATOR_SDK_VERSION=${OPERATOR_SDK_VERSION:-v1.27.0}
 declare -r YQ_VERSION=${YQ_VERSION:-v4.34.2}
 declare -r CRDOC_VERSION=${CRDOC_VERSION:-v0.6.2}
+declare -r OC_VERSION=${OC_VERSION:-4.13.0}
 
 # install
 declare -r KUSTOMIZE_INSTALL_SCRIPT="https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh"
 declare -r OPERATOR_SDK_INSTALL="https://github.com/operator-framework/operator-sdk/releases/download/$OPERATOR_SDK_VERSION/operator-sdk_${GOOS}_${GOARCH}"
 declare -r YQ_INSTALL="https://github.com/mikefarah/yq/releases/download/$YQ_VERSION/yq_${GOOS}_${GOARCH}"
+declare -r OC_INSTALL="https://mirror.openshift.com/pub/openshift-v4/clients/ocp/$OC_VERSION"
```

Comment:
```suggestion
declare -r OC_URL="https://mirror.openshift.com/pub/openshift-v4/clients/ocp/$OC_VERSION"
```
 is OC_VERSION defined?

Reply:
Yes we are defining [here](https://github.com/sustainable-computing-io/kepler-operator/blob/d04ae6534e458c60a70ac97a76e8b979c5f50722/hack/tools.sh#L36)